packages/mobile/src/components/core/Screen/ScreenHeaderButton.tsx

```diff
@@ -1,10 +1,14 @@
-import type { ButtonProps } from 'app/components/core'
-import { Button } from 'app/components/core'
+import { View } from 'react-native'
 
-type ScreenHeaderButtonProps = ButtonProps & {
-  title: string
-}
+import { SelectablePill } from '@audius/harmony-native'
+import type { SelectablePillProps } from 'app/harmony-native/components/input/SelectablePill/types'
+
+type ScreenHeaderButtonProps = Omit<SelectablePillProps, 'type'>
 
 export const ScreenHeaderButton = (props: ScreenHeaderButtonProps) => {
-  return <Button key={props.title} variant='secondary' size='xs' {...props} />
+  return (
+    <View>
```

Comment:
What's the extra View for here?

Reply:
Supremely weird, but the parent views align-items "center" was not applying to the button, so i added this view so the title + filter buttons in the header like feed was aligned correctly. will add a comment here though :)